x-pack/plugins/security_solution/common/api/timeline/model/api.ts

```diff
@@ -45,6 +45,16 @@ export type BareNote = runtimeTypes.TypeOf<typeof BareNoteSchema>;
  */
 export type BareNoteWithoutExternalRefs = Omit<BareNote, 'timelineId'>;
 
+export const BareNoteWithoutExternalRefsSchema = runtimeTypes.partial({
```

Comment:
If you can add this to the models here `x-pack/plugins/security_solution/common/api/timeline/model/components.yaml`. May need to update the route schemas that make use of this as well